core/backend/package.json

```diff
@@ -6,7 +6,7 @@
   "typings": "lib/cjs/core-backend",
   "license": "MIT",
   "engines": {
-    "node": ">=16.13.0 < 19.0"
+    "node": "^18.0.0"
```

Comment:
maybe we should actually set the engine to be ^18.12.0; which is what we're saying is the minimum compatible version in some of the documentation, and the testing pipelines

Reply:
Used [this discussion](https://github.com/iTwin/itwinjs-core/pull/4295#discussion_r1005113967) as a reference to what range this should be set. In docs, used `^18.12.0` to encourage using LTS versions. 

`^18.12.0` would make thinks more consistent.

Reply:
thx for bringing that back up, i guess based on calebs comment we should leave as is then